src/toil/test/provisioners/aws/awsProvisionerTest.py

```diff
@@ -66,7 +66,9 @@ def cleanJobStoreUtil(self):
     def __init__(self, methodName):
         super(AbstractAWSAutoscaleTest, self).__init__(methodName=methodName)
         self.instanceType = 'm3.large'
-        self.keyName = 'jenkins@jenkins-master'
+        self.keyName = os.getenv('TOIL_AWS_KEYNAME')
+        if not self.keyName:
+            self.fail("TOIL_AWS_KEYNAME not set.")
```

Comment:
Perhaps we should give more info on what this environment variable does so that if someone naively runs this test and gets this error, they will know what to do make the tests run.

Also maybe skipping the test makes more sense than failing?